packages/dapp/src/components/Main.tsx

```diff
@@ -158,6 +162,11 @@ export const Main: React.FunctionComponent = () => {
           <Route path={routes.SUBMIT_CHALLENGE} component={AsyncComponent(SubmitChallengePage)} />
           <Route path={routes.SUBMIT_APPEAL_CHALLENGE} component={AsyncComponent(SubmitAppealChallengePage)} />
           <Route path={routes.REQUEST_APPEAL} component={AsyncComponent(RequestAppealPage)} />
+          <Route path={routes.LISTING_STORY_BOOST_PAYMENT} component={AsyncComponent(Listing, { payment: true })} />
+          <Route
+            path={routes.LISTING_STORY_BOOST_NEWSROOM}
+            component={AsyncComponent(Listing, { newsroomDetails: true })}
+          />
```

Comment:
If the newsroom modal is blocked from rendering on listing page you don't need this.